Sources/SwiftGRPCNIO/HTTP1ToRawGRPCServerCodec.swift

```diff
@@ -16,6 +17,12 @@ public enum RawGRPCServerResponsePart {
   case status(GRPCStatus)
 }
 
+enum ContentType: String {
+  case binary = "application/grpc"
```

Comment:
Please add headerdocs for the enum overall and the individual cases, maybe with a short explanation of the different stream semantics.

Reply:
Done.